tests/features/multisite_wysiwyg/rich_text_editing.feature

```diff
@@ -0,0 +1,26 @@
+@api
+Feature: Testing the rich text options available with the toolbar present on
+  WYSIWYG fields like "Body".
+  I want to be able to format my content with rich text options supplied by the
+  displayed toolbar
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+
+  @javascript
+  Scenario: I create a content that contains a link with the following tag
+  attributes: id, class and hreflang
```

Comment:
When possible I try to respect the rule on the max. of characters per line (80 per line).